train_controlnet3d.py

```diff
@@ -552,7 +550,7 @@ def collate_fn(examples):
     conditioning_pixel_values = torch.stack([example["conditioning_pixel_values"] for example in examples])
     conditioning_pixel_values = conditioning_pixel_values.to(memory_format=torch.contiguous_format).float()
 
-    input_ids = torch.stack([example["input_ids"] for example in examples])
+    input_ids = torch.cat([example["input_ids"] for example in examples])
```

Comment:
Does this fix the issue I had https://github.com/JCBrouwer/ControlNet3D/compare/main...runners#diff-4bfaa9cca41e6a61f039d45b1b4acd9a9cb6f0385245e7598635649897d2a5efR893

Reply:
I'm not super clear which line the link is referring to, but if it's line 912, then yes. This ensures we don't get an extra unit dimension which we don't need.